src/web/routes/application/postcode/adapters.js

```diff
@@ -0,0 +1,26 @@
+const { compose, map, pick, prop, propOr } = require('ramda')
+
+const RESULTS_PROP = 'results'
+
+const DELIVERY_POINT_ADDRESS_PROP = 'DPA'
+
+const ADDRESS_FIELDS = [
+  'ADDRESS',
+  'ORGANISATION_NAME',
+  'BUILDING_NUMBER',
+  'SUB_BUILDING_NAME',
+  'BUILDING_NAME',
+  'THOROUGHFARE_NAME',
```

Comment:
We should probably include DEPENDENT_THOROUGHFARE_NAME as well - CH53QW

Reply:
Added this one.